From 21aecb6eb07eae03bfae851547b3fc473cc1a066 Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Wed, 7 Oct 2015 16:55:14 -0700
Subject: [PATCH] Laird: WB50N: remove mtd partitions from bootargs

Having the mtd partitions in the bootargs screws up the define in the device
tree. We should let the dts be authoritative here.
Also, remove redundant and conflicting bootargs.

Bug: 8370
---
 include/configs/wb50n.h | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/include/configs/wb50n.h b/include/configs/wb50n.h
index f5d7c6f..8cea68c 100644
--- a/include/configs/wb50n.h
+++ b/include/configs/wb50n.h
@@ -186,13 +186,7 @@
 
 
 #define CONFIG_BOOTARGS							\
-	"console=ttyS0,115200 earlyprintk loglevel=4 rw mem=64M "				\
-	"mtdparts=atmel_nand:128k(bootstrap),512k(uboot),"		\
-	"128k(u-boot-env),128k(redund-env),"			\
-	"2560k(kernel-a),2560k(kernel-b),"				\
-	"38912k(rootfs-a),38912k(rootfs-b),"			\
-	"46208K(user),-(logs) "							\
-	"rootfstype=ubifs ubi.mtd=6 root=ubi0:rootfs"
+	"rw rootfstype=ubifs ubi.mtd=6 root=ubi0:rootfs"
 
 #define CONFIG_BAUDRATE			115200
 
-- 
1.9.1

